pom.xml

```diff
@@ -456,16 +456,13 @@
                   <arg value="-DCMAKE_CUDA_ARCHITECTURES=${GPU_ARCHS}"/>
                   <arg value="-DCMAKE_INSTALL_PREFIX=${libcudf.install.path}"/>
                   <arg value="-DCUDF_DEPENDENCY_PIN_MODE=${libcudf.dependency.mode}"/>
+                  <arg value="-DCUDA_STATIC_CUFILE=ON"/>
                   <arg value="-DCUDA_STATIC_RUNTIME=ON"/>
-                  <arg value="-DCUDF_ENABLE_ARROW_S3=OFF"/>
-                  <arg value="-DCUDF_ENABLE_ARROW_PARQUET=ON"/>
```

Comment:
Is this always on then? I added this so that our native footer parser would work. CUDF before was explicitly removing it because it wasted space for something that they didn't want to use.

Reply:
cudf doesn't have Arrow anymore, and this setting was removed in rapidsai/cudf#16640 and moved to the cudfjni CMakeLists.txt as hardcoded to on here: https://github.com/rapidsai/cudf/blob/branch-24.12/java/src/main/native/CMakeLists.txt#L215